google-http-client/src/main/java/com/google/api/client/http/HttpHeaders.java

```diff
@@ -811,6 +815,35 @@ public HttpHeaders setAuthenticate(String authenticate) {
     return this;
   }
 
+  /**
+   * Sets the {@code "Warning"} header or {@code null} for none.
+   *
+   * <p>Overriding is only supported for the purpose of calling the super implementation and
+   * changing the return type, but nothing else.
+   *
+   * @since 1.28
+   */
+  public HttpHeaders setWarning(String warning) {
```

Comment:
Looks like `addWarning` would be more accurate than `setWarning`.

Reply:
@elharo , thanks for the comment, renamed method.